src/Scene/NodeProcess.js

```diff
@@ -15,11 +13,8 @@ import { l_ELEVATION, l_COLOR } from 'Renderer/LayeredMaterial';
 
 export const SSE_SUBDIVISION_THRESHOLD = 6.0;
 
-function NodeProcess(camera, ellipsoid, bbox) {
+function NodeProcess(ellipsoid) {
```

Comment:
IMHO: this class is `GlobeTileProcess`.
Maybe later we'll be able to extract some generic codes from this, but currently it only serves 1 purpose: process a Tile from a Globe.